CMakeLists.txt

```diff
@@ -145,6 +145,11 @@ if (CMAKE_C_COMPILER_ID MATCHES "(Clang|Gnu|GNU)")
   set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall")
 endif()
 
+# set rsync_EXPORTS on WIN32
+if(WIN32)
+  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Drsync_EXPORTS")
+endif()
```

Comment:
I was under the impression cmake defines this automatically;

https://stackoverflow.com/questions/27429732/cmake-adds-dlibname-exports-compile-definition

Reply:
You're right, it does; however the build failures are due to the different targets such as `sumset_test` and `rdiff` not having `rsyncs_EXPORTS` defined for them.